libmamba/src/core/package_info.cpp

```diff
@@ -110,7 +110,7 @@ namespace mamba
             build_number = n;
         }
 
-        Id real_repo_key = pool_str2id(pool, "solvable:real_repo_url", 1);
+        static Id real_repo_key = pool_str2id(pool, "solvable:real_repo_url", 1);
```

Comment:
Notice that this would make this constructor not threadsafe (I don't know if that can be a problem in the future).

Reply:
I think we should really define these variables in the MPool constructor so that we have them available as keys everywhere. I haven't figured out how to do it properly yet.

Reply:
I think we can ignore this for now since I think any operations on the Pool / Repo in libsolv aren't really threadsafe.